sites/svelte-5-preview/src/routes/docs/content/01-api/02-runes.md

```diff
@@ -490,14 +490,34 @@ let { a, b, c, ...everythingElse }: MyProps = $props();
 >
 > ...TypeScript [widens the type](https://www.typescriptlang.org/play?#code/CYUwxgNghgTiAEAzArgOzAFwJYHtXwBIAHGHIgZwB4AVeAXnilQE8A+ACgEoAueagbgBQgiCAzwA3vAAe9eABYATPAC+c4qQqUp03uQwwsqAOaqOnIfCsB6a-AB6AfiA) of `x` to be `string | number`, instead of erroring.
 
-Props cannot be mutated, unless the parent component uses `bind:`. During development, attempts to mutate props will result in an error.
+By default props are treated as readonly, meaning reassignments will not propagate upwards and mutations will result in a warning at runtime in development mode. You will also get a runtime error when trying to `bind:` to a readonly prop in a parent component. To declare props as bindable, use [`$bindable()`](#bindable).
```

Comment:
In pure Markdown the link appears to work, in the generated docs the ID of the heading is [`$props-$bindable`](https://svelte-5-preview.vercel.app/docs/runes#$props-$bindable), though, and this link is not adjusted.

All the links on the right side also generate this alert for when you navigate from the REPL to the docs:
![image](https://github.com/sveltejs/svelte/assets/834235/6cd18b5d-a050-494c-82a6-ac5c81944daa)